code/renderers/vue3/src/globals.ts

```diff
@@ -3,3 +3,5 @@ import { global } from '@storybook/global';
 const { window: globalWindow } = global;
 
 globalWindow.STORYBOOK_ENV = 'vue3';
+globalWindow.STORYBOOK_VUE_GLOBAL_PLUGINS ||= [];
+globalWindow.STORYBOOK_VUE_GLOBAL_MIXINS ||= [];
```

Comment:
I don't see those 2 being used anywhere.

Reply:
not yet used I planned you used them in the begining but i'm still thinking of the right implementation